Support backup and restore with ObjectBucketName set in environment variables (#637)
cybozu-go/moco: docs/usage.md

...
```

> >**Note:** If you want to specify the ObjectBucket name in configMap or secret, you can use envFrom and specify environments variable in `ObjectBucketName` like below.

Reviewer: Do you mean the below?
```suggestion
>**Note:** If you want to specify the ObjectBucket name in a ConfigMap or Secret, you can use `envFrom` and specify the environment variable name in `jobConfig.bucketConfig.bucketName` as follows.
```

Author: @ymmt2005 
Yes, thank you for pointing that out.

>This behavior is tested.

```yaml
apiVersion: moco.cybozu.com/v1beta2
kind: BackupPolicy
metadata:
  namespace: backup
  name: daily
spec:
  jobConfig:
    bucketConfig:
      bucketName: "$(BUCKET_NAME)"
      endpointURL: http://minio.default.svc:9000
      usePathStyle: true
    envFrom:
    - configMapRef:
        name: bucket-name

> ...

Reviewer: It'd be more helpful if we include an example ConfigMap.
```yaml
...
---
apiVersion: v1
kind: ConfigMap
data:
  BUCKET_NAME: foobar
